packages/notebook/src/panel.ts

```diff
@@ -133,16 +133,18 @@ export class NotebookPanel extends DocumentWidget<Notebook, INotebookModel> {
     sender: any,
     args: Session.IKernelChangedArgs
   ): void {
-    if (!this.model || !args.newValue) {
+    const kernel = args && args.newValue;
```

Comment:
You shouldn't need the `args &&`, right?

Reply:
You do, because the error that led me here was when `args` is `null`.

Reply:
When is this signal being emitted with null?

Reply:
https://github.com/jupyterlab/jupyterlab/blob/master/packages/apputils/src/clientsession.tsx#L606

https://github.com/jupyterlab/jupyterlab/blob/master/packages/apputils/src/clientsession.tsx#L721

Reply:
I think we should fix those places instead.

Reply:
I suppose strictNullChecks would have caught this in the original PR.

Reply:
I assumed those were correct. Should they emit `{ oldValue: null, newValue: null }` instead?

Reply:
Yes, I think that is right. If the kernel is disposed, we emit a null elsewhere, and we don't have a new value, so that should be null too.

Reply:
Thanks for catching this!

Reply:
Cool, I'll update!